### ISSUE TITLE
Inconsistent results between grid/text displays

### ISSUE THREAD

**Issue by alper-sunar** [dbeaver/dbeaver#16673]
#### System information: 
- Operating system (distribution) and version: MacOS 11.6
- DBeaver version: 22.0.5.202205220912

#### Connection specification:
- Database name and version: PostgreSQL 13.6 on x86_64-pc-linux-gnu, compiled by Debian clang version 12.0.1, 64-bit
- Driver name: PostgreSQL standard driver
- Do you use tunnels or proxies (SSH, SOCKS, etc)? No 

#### Describe the problem you're observing:
Inconsistent display between grid and text results.

#### Steps to reproduce, if exist:
```
CREATE TABLE display_test (
	bid_rate float4 NULL

);

INSERT INTO display_test (
"bid_rate"
      )
      SELECT
"bid_rate"
      FROM
         json_populate_recordset (NULL::display_test, 
  '[{"bid_rate":46.25}, {"bid_rate":46.45}]')
  
SELECT bid_rate FROM display_test
```

Switch between grid/text results.
Expected: Grid result should show 46.45
Actual: Grid result shows 46.450000763

![Dbeaver](https://user-images.githubusercontent.com/92756049/171516349-45328646-a024-4536-a71b-14446854e75e.gif)

**Comment by LonwoLonwo** [comment]
Hello @alper-sunar 

You can change the number of fractions digits in the Preferences -> Editors -> Data Editor -> Data Formats.
 
![2022-06-02 10_43_03-Preferences](https://user-images.githubusercontent.com/45152336/171581110-3879abe5-d675-4eea-a657-4bccc7318068.png)

But anyway, this situation looks buggy. Thanks for the report.

**Comment by LonwoLonwo** [comment]
This situation is also incorrect

float4 without modifiers = double precision, but float(2) has precision 2, not 10.

![2022-06-02 10_56_08-DBeaver Ultimate 22 1 0 - _postgres_ Script-32](https://user-images.githubusercontent.com/45152336/171582429-b794b999-5909-48b2-9dab-0c3deea4ba33.png)

**Comment by alper-sunar** [comment]
Agreed, even if it can be managed with settings, the display should be consistent between different values imho. Thank you.

**Comment by serge-rider** [comment]
This behaviour is controlled by "display format" setting.
For plain text it is "Edit" (row value without numeric formatting), for grid it is "Display" (numeric formatting is applied).
For plaintext it can be changed in data editor preferences.

Now I unified this. Display format can be changed for both grid and text presentations.
It was also added to the context menu:
![image](https://user-images.githubusercontent.com/6398845/194952488-fdbe2946-0272-4139-9738-3fae44eaade5.png)
![image](https://user-images.githubusercontent.com/6398845/194952565-b00f0569-43a5-434f-a7e8-4907135c67d2.png)
